app/models/miq_server/update_management.rb

```diff
@@ -63,6 +63,8 @@ def attempt_registration
     configure_yum_proxy
     # HACK: #enable_repos is not always successful immediately after #attach_products, retry to ensure they are enabled.
     5.times { repos_enabled? ? break : enable_repos }
+    update_attributes(:upgrade_message => "Registration process completed successfully")
```

Comment:
Should this just be "registration successful" like [here](https://github.com/ManageIQ/manageiq/pull/17340/files#diff-7caa3de16af726f5ad141196450f637dR96)? Or are these conditions different?

Reply:
No.  I wanted to differentiate the completion of "subscription-manager register" and the completion of the entire process of registering (register, attach, enable repos).